backend/src/main/java/rocks/inspectit/gepard/agentmanager/configuration/service/ConfigurationService.java

```diff
@@ -0,0 +1,19 @@
+/* (C) 2024 */
+package rocks.inspectit.gepard.agentmanager.configuration.service;
+
+import org.springframework.stereotype.Service;
+import rocks.inspectit.gepard.agentmanager.configuration.model.InspectitConfiguration;
+
+@Service
+public class ConfigurationService {
+
+  private InspectitConfiguration inspectitConfiguration;
```

Comment:
I would recommend to make the configuraiton `volatile` should it's faster available for other threads if updated.

Reply:
Done.